src/components/organisms/UiMenu/stories/HasButton.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <UiPopover class="has-button">
+    <UiMenu
+      v-bind="args"
+      :items="itemsToRender"
+      role="listbox"
+      aria-label="Chronic condition"
+      aria-multiselectable="true"
+      aria-activedescendant=""
```

Comment:
![image](https://github.com/user-attachments/assets/e67cb2cc-8935-4c39-9fe5-a5fcc337fc1c)

`aria-activedescendant="ID_REF"` is empty and without `dash` and never changes its value. Why?